recipes/xenon/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "xenon" %}
+{% set version = "0.5.3" %}
+{% set sha256 = "edd793ff6a5b4af9c56899e3109285870bfdefaf8cb9cd4a6abca17d192bfbc4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - xenon = xenon:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - radon
+    - requests >=2.0
+    - pyaml >=3.11
+    - python
+
+test:
+  imports:
+    - xenon
+  commands:
+    - xenon -v
+
+about:
+  home: https://github.com/rubik/xenon
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Did you open an issue upstream about including the License in source distributions?

Reply:
Done:
https://github.com/rubik/xenon/issues/20